frontend/src/components/Secrets/GSecretDialogAlicloud.vue

```diff
@@ -8,7 +8,7 @@ SPDX-License-Identifier: Apache-2.0
   <g-secret-dialog
     v-model="visible"
     :data="secretData"
-    :data-valid="valid"
+    :secret-validations="v$"
```

Comment:
I have already said that I don't like the this pattern to pass the full vuelidate object to a subcomponent. I don't understand why this is needed and this is the onlt way to solve the issue.

Reply:
removed component